HISTORY.rst

```diff
@@ -3,6 +3,22 @@
 Release History
 ===============
 
+
+0.14.0
++++++++++++++++
+
+**Digital Twin updates**
+
+* Added `az dt data-history` command group, which will allow users to configure a data history connection
```

Comment:
We'll have to adjust some things to get these changes merged for a near-term release.  Like not exposing the `az dt data-history` to the user.